Remotion/Web/Core/Res/Themes/NovaGray/HTML/StyleUtility.js

```diff
@@ -26,31 +26,31 @@ StyleUtility.CreateBorderSpans = function (selector)
 
 StyleUtility.AddBrowserSwitch = function ()
 {
-  var browser;
-  if (navigator.appVersion.indexOf ('Edge') !== -1)
+  function searchBrowser(brands, search)
   {
-    browser = 'edge';
+    return brands.some(c => c.brand === search);
   }
-  else if (navigator.appVersion.indexOf ('Chrome') !== -1)
-  {
-    browser = 'blink';
-  }
-  else if (navigator.appVersion.indexOf ('OPR') !== -1)
-  {
-    browser = 'blink';
-  }
-  else if (navigator.appVersion.indexOf('WebKit') !== -1)
-  {
-    browser = 'webkit';
-  }
-  else if (navigator.appName === 'Netscape')
+
+  let browser;
+  if (navigator.userAgentData)
   {
-    browser = 'mozilla';
+    const brands = navigator.userAgentData.brands;
+    if (searchBrowser(brands, 'Microsoft Edge') || searchBrowser(brands, 'Google Chrome') || searchBrowser(brands, 'Opera'))
+      browser = 'blink';
```

Comment:
readability:
I would prefer separate if/elseif here because it makes it easier to follow (shorter lines). if you additionally want to prevent the duplication of the string-value, you can define a `const var blink='blink'` and use that during the assignment.